docs/src/main/asciidoc/cache.adoc

```diff
@@ -522,6 +522,67 @@ quarkus.cache.caffeine."bar".maximum-size=1000 <2>
 <1> The `foo` cache is being configured.
 <2> The `bar` cache is being configured.
 
+== Enabling Micrometer metrics
+
+Each cache declared using the <<#annotations-api,annotations caching API>> can be monitored using Micrometer metrics.
+
+[NOTE]
+====
+The cache metrics collection will only work if your application depends on a `quarkus-micrometer-registry-*` extension.
+See the <<micrometer.adoc,Micrometer metrics guide>> to learn how to use Micrometer in Quarkus.
```

Comment:
```suggestion
See the xref:micrometer.adoc[Micrometer metrics guide] to learn how to use Micrometer in Quarkus.
```
let's be consistent with how I did things elsewhere for cross guide links.

Reply:
Fixed, thanks!